### DISCUSSION TITLE
Disable REI overlay on some screens ? (registerBounds unavailable)

### DISCUSSION THREAD

**Discussion by nebspacefarer** [shedaniel/RoughlyEnoughItems#986] [Q & A] [answered]
Hello,

I am trying to disable REI on my custom screen from my mod. I cannot find the `registerBounds()` method as shown on the Wiki anywhere. Wiki hasn't been updated for a long time it seems, so I guess the API must have changed since then.

Any pointer on how it should be done? I want to completely hide REI as it is not a screen that offers recipes use. I have a custom ScreenHandler and a HandledScreen.
Thank you!

**Comment by shedaniel** [comment, marked as answer]
it is now registerScreens, then you will want to register a decider that hides rei https://github.com/shedaniel/RoughlyEnoughItems/blob/ed1cae3952315022b2928d264a9a98e57bc33bcc/api/src/main/java/me/shedaniel/rei/api/client/registry/screen/ScreenRegistry.java#L96
